resources/default.rb

```diff
@@ -43,7 +43,7 @@
 # Lowering upgrade_delay limit is not recommended.
 attribute :upgrade_delay, kind_of: Integer, default: 60
 attribute :product_name, kind_of: String, default: 'chef'
-attribute :rubygems_url, kind_of: String
+attribute :rubygems_url, kind_of: Symbol, default: lazy { Chef::Config[:rubygems_url] }
```

Comment:
Just verified in chef-shell on 14.13.11 but `Chef::Config[:rubygems_url]` returns a string. Setting the kind_of to Symbol breaks this cookbook.

Reply:
```Chef::Exceptions::ValidationFailed: Option rubygems_url must be a kind of [Symbol]!  You passed "https://www.rubygems.org".```

Reply:
whoops, thx for catching that